packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -80,21 +87,29 @@ export const publishInterchainAssetFromBank = async (
       assetKind: AssetKind.NAT,
     },
   };
-  const { creatorFacet: mintP, publicFacet: issuerP } = E.get(
-    E(zoe).startInstance(mintHolder, {}, terms, undefined, keyword),
-  );
 
-  const [issuer, brand] = await Promise.all([issuerP, E(issuerP).getBrand()]);
-  const kit = { mint: mintP, issuer, brand };
+  const startUpgradeable = await startUpgradeableP;
+  produceVBankKits.resolve([]);
```

Comment:
why does it now need to produce `vBankKits` and is an empty list correct?

Reply:
Yes, we resolve it to an empty list initially (if it's already initialized then this is a no-op). Then in `produceResults` we append each kit to the list, saving it.

Reply:
ah, I didn't grok that in my first read